powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/controller/api/OperationTemplatesController.java

```diff
@@ -106,10 +117,13 @@ public ObjectResponse<OperationTemplateDetailResponse> createOperationTemplate(@
      * @throws Exception In case the service throws exception.
      */
     @PostMapping("/update")
-    public ObjectResponse<OperationTemplateDetailResponse> updateOperationTemplate(@RequestBody ObjectRequest<OperationTemplateUpdateRequest> request) throws Exception {
-        logger.info("OperationTemplateUpdateRequest call received: {}", request);
-        final ObjectResponse<OperationTemplateDetailResponse> response = new ObjectResponse<>(service.updateOperationTemplate(request.getRequestObject()));
-        logger.info("OperationTemplateUpdateRequest succeeded: {}", response);
+    public ObjectResponse<OperationTemplateDetailResponse> updateOperationTemplate(@Valid @RequestBody ObjectRequest<OperationTemplateUpdateRequest> request) throws Exception {
+        final OperationTemplateUpdateRequest req = request.getRequestObject();
+        logger.info("action: updateOperationTemplate, state: initiated, id: {}, operationType: {}", req.getId(), req.getOperationType());
```

Comment:
When passing an id, the type seems redundant.
```suggestion
        logger.info("action: updateOperationTemplate, state: initiated, templateId: {}", req.getId());
```